support-frontend/test/selenium/util/PostDeployTestUser.scala

```diff
@@ -24,3 +28,31 @@ class PostDeployTestUser(driverConfig: DriverConfig) extends TestUser {
   val username = testUsers.generate()
   addTestUserCookies(username)
 }
+
+class PostDeployTestUserSubs(driverConfig: DriverConfig) extends TestUser {
+
+  private val client = new OkHttpClient()
+  private val requestToIdapi = new Request.Builder()
+    .url(Config.idapiNewTestUserUrl)
+    .addHeader(Config.idapiClientAccessTokenName, Config.idapiClientAccessTokenSecret)
+    .post(RequestBody.create(MediaType.parse("application/json"), "{}"))
+    .build()
+  private val response = client.newCall(requestToIdapi).execute()
+  private val responseBodyAsJson = parse(response.body().string()).getOrElse(Json.Null)
+  private val cursor = responseBodyAsJson.hcursor
```

Comment:
I would probably pull the identity call out into a separate class that returns the cookie values in a `List[(String, String)]` as a minimum.
You can then check the response code and the json structure in there, and either throw a suitable exception (or even put it into an `Either`) so that we can diagnose things when they go wrong.
As it is, we might just throw NoSuchElementException on line 50/51 and it wouldn't be clear what went wrong.
Ideally we can have enough information to decide is the a bug or a transient identity issue and enough to raise an issue with Identity team without having to run them locally.

Reply:
Thank you! Addressed this in the new commit, do let me know if it's better.